getting-started/validate.md

```diff
@@ -4,33 +4,35 @@ As you begin to add solution APIs to your mobile implementation, you are going t
+Once you have Griffon integrated, you can create a [new session](https://aep-sdks.gitbook.io/docs/beta/project-griffon/using-project-griffon#creating-sessions) by either scanning a QR code or by following a unique deep link URL.
 
 The main interface for Griffon will show a running event list of all SDK events, including a configuration response event that will provide a readout of all configuration values obtained from Adobe Experience Platform Launch.
 
 ![Project Griffon Configuration Response](../.gitbook/assets/configurationresponse.png)
 
-## Adobe Analytics View
+## Adobe Analytics view
 
-The [Adobe Analytics](../beta/project-griffon/using-project-griffon/adobe-analytics-and-project-griffon.md#using-project-griffon-for-adobe-analytics) event list offers a focused view of just analytics events triggered in the application. You can easily sort through all of the track action and track state calls. What is unique to the [analytics view in Griffon](../beta/project-griffon/using-project-griffon/adobe-analytics-and-project-griffon.md) is the ability to see both the raw hit request sent to analytics and the post-processed details.
+The [Adobe Analytics](../beta/project-griffon/using-project-griffon/adobe-analytics-and-project-griffon.md#using-project-griffon-for-adobe-analytics) event list offers a focused view of analytics events triggered in the application. You can sort through all of the track action and track state calls. In the [analytics view in Griffon](../beta/project-griffon/using-project-griffon/adobe-analytics-and-project-griffon.md), you can see both the raw hit request sent to Analytics and the post-processed details.
 
-![Project Griffon Analytics View](../.gitbook/assets/GriffonAnalytics.png)
+![Project Griffon analytics view](../.gitbook/assets/GriffonAnalytics.png)
 
-## Places Service View
+## Places Service view
 
-Testing and validating actions based on geofence data can be incredibly difficult and time consuming. Project Griffon helps ease the pain in testing point of interest entries and exits. The Places event list provides a focused view showing all events related to Places Service, including user authorization level granted and requests for nearby points of interest \(POIs\). While a device is connected to an active Griffon session the map view will show a timeline of POI entries and exits. If you would like to test actions or experiences triggered by geofence entries and exits, the map view will allow you to spoof or simulate your location by clicking on any area within the map.
+Project Griffon simplifies testing point-of-interest entries and exits. The Places event list provides a focused view showing all events related to Places Service, including user authorization level granted and requests for nearby points of interest (POIs). 
```

Comment:
@calebk1, we have hyphenated "points of interest" in the first sentence, but not the second. Both are referring to the same thing. We call them "points of interest" or "POIs".

Reply:
Yeah, it should be all un-hyphenated. I'll fix that!